ofl/mochiypopone/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,3 @@
+<p>Mochiypop is a casual Gothic font created based on characters written by girls in their 20s in Japan.</p>
+<p>A cute display font, you can use it widely, such as for manga, magazines, movies, and signs.</p>
+<p>The included Kanji are modified versions of Source Han Sans.</p>
```

Comment:
* The description should have a final paragraph like many do, linking to the upstream repo. 

* The description should link to sibling families. 

The cabin and Baloo 2 sets of families do those 2 things, as I recall. 

* We call it Noto Sans JP around these parts ;)

* I prefer tags on their own line, one sentence per line (no hand wraps), and a blank link as a final line; this makes diffs easier to read on GitHub and makes Unix piping better

Reply:
Done (I think)